### PR TITLE
Add a11y test for all content

### PR DESCRIPTION
**Pull request by smoralesd**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000 **N/A**
- [x] Added relevant unit test for your changes. (`npm run test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`npm run precheckin`) **N/A**
- [ ] Added screenshots/GIFs for UI changes. **N/A**

#### Description of changes

Adding automated a11y test for all the content we have under src/content folder. Any content that we add there will be automatically tested for accessibility issues. Test run on normal mode as well as high contrast mode.

#### Notes for reviewers

There is of course an increase in the running time for e2e tests now.

### CODE REVIEW

File: src/DetailsView/components/generic-toggle.tsx [hunk 1]
```diff
@@ -12,28 +13,26 @@ export interface GenericToggleProps {
...
-                        onText={'On'}
-                        offText={'Off'}
-                        ariaLabel={this.props.name}
-                    />
-                </div>
-                <div className={'toggle-description'}>{this.props.description}</div>
-            </div>
-        );
-    }
+export const GenericToggle = NamedSFC<GenericToggleProps>('GenericToggle', props => {
```
Comment (Shobhit1) [inline src/DetailsView/components/generic-toggle.tsx]: I love this.

File: src/tests/end-to-end/tests/content/a11y-content.test.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            await browser.close();
+            browser = undefined;
+        }
+    });
+
+    const contentPaths = contentPages.allPaths();
+
+    it.each(contentPaths)('%s', async path => {
+        const content = await browser.newContentPage(path);
+
```
Comment (devaradhanm) [inline src/tests/end-to-end/tests/content/a11y-content.test.ts]: you could also add test for hc theme

Comment (smoralesd) [reply src/tests/end-to-end/tests/content/a11y-content.test.ts]: out of scope as mention on **Notes for reviewers**

Comment (smoralesd) [reply src/tests/end-to-end/tests/content/a11y-content.test.ts]: @devaradhanm I had some more time today and end up adding tests for HC mode too.

**Pull request merged**